requirements.txt

```diff
@@ -18,3 +18,4 @@ sphinx>=4.4.0
 sphinx-rtd-theme>=1.0.0
 myst-parser>=0.17.0
 twine>=3.8.0
+dohq-artifactory==0.8.1
```

Comment:
I happened to be using a python 3.11 virtual environment when running `python package.py` and got:
```
⏚ [bryan:~/dev/seeq-udf-ui] [.venv] [ruby-2.5.1] task/kcbl/package-udf-for-add-on-CRAB-39458 11s ± python package.py
Traceback (most recent call last):
  File "/Users/bryan/dev/seeq-udf-ui/package.py", line 6, in <module>
    from artifactory import ArtifactoryPath
  File "/Users/bryan/dev/seeq-udf-ui/.venv/lib/python3.11/site-packages/artifactory.py", line 640, in <module>
    class _ArtifactoryAccessor(pathlib._Accessor):
                               ^^^^^^^^^^^^^^^^^
AttributeError: module 'pathlib' has no attribute '_Accessor'
```
Using version `0.9.1` worked.

Reply:
Ah this is because of importing Artifactory. We didn't end up needing it though so I'll take that off.